aerogear/mobile-docs
# update nav and add container file

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8019

## What
Adding in nav changes and placeholder files

## Why
To prepare for upcoming docs work

## Verification Steps
1. Follow the steps 1 and 2 [here](https://github.com/aerogear/mobile-docs#publishing) to build it
2. Verify the navigation works
 
## Checklist:
